server/dive_tasks/tasks.py

```diff
@@ -329,11 +329,12 @@ def train_pipeline(
             groundtruth_path = organize_folder_for_training(
                 download_path, download_path / groundtruth["name"]
             )
-            input_groundtruth_list.append((download_path, groundtruth_path))
             # Download input media
             input_media_list = download_source_media(gc, source_folder, download_path)
             if fromMeta(source_folder, TypeMarker) == VideoType:
                 download_path = Path(input_media_list[0])
+            # Set media source location
+            input_groundtruth_list.append((download_path, groundtruth_path))
```

Comment:
Fix ordering where the wrong `download_path` was being fed into `input_groundtruth_list`